packs/kubedoom-1.0.0/pack.json

```diff
@@ -0,0 +1,13 @@
+{
+  "addonType": "app services",
+  "annotations": {
+    "source": "community",
+    "contributor": "spectrocloud"
+  },
+  "cloudTypes": ["all"],
```

Comment:
@lennessyy should we just make this for edge versus all? Or, you can improve the logic and provide a present for edge and one for not edge. The non-edge preset could deploy a load balancer.

Your take but if you only want this for edge then it's best make it edge only.